src/utils/pointer-move-event-input.js

```diff
@@ -0,0 +1,40 @@
+import {PointerEventInput} from 'hammerjs';
+
+const POINTER_MOVE = 'pointermove';
+
+// Copied from Hammer.js' pointerevent.js
+const IE10_POINTER_TYPE_ENUM = {
```

Comment:
@ibgreen @ericsoco Yes, we should only support IE11. IE10 doesn't support WebGL.